pkg/inspector/image-inspector.go

```diff
@@ -93,7 +93,7 @@ func (i *defaultImageInspector) Inspect() error {
 	container, err := client.CreateContainer(docker.CreateContainerOptions{
 		Name: randomName,
 		Config: &docker.Config{
-			Image:      i.opts.Image,
+			Image: i.opts.Image,
```

Comment:
I am sorry, Its a problem with my vim and its lint.. its autocorrects this. I will look into this.. Oo